pkg/cmd/attestation/verification/extensions.go

```diff
@@ -46,5 +51,26 @@ func verifyCertExtensions(attestation *AttestationProcessingResult, tenant, owne
 		}
 	}
 
+	// if issuer is anything other than the default, use the user-provided value;
+	// otherwise, select the appropriate default based on the tenant
+	if issuer != GitHubOIDCIssuer {
+		want = issuer
+	} else {
+		if tenant != "" {
+			want = fmt.Sprintf(GitHubTenantOIDCIssuer, tenant)
+		} else {
+			want = GitHubOIDCIssuer
+		}
+	}
+
+	certIssuer := attestation.VerificationResult.Signature.Certificate.Extensions.Issuer
+	if !strings.EqualFold(want, certIssuer) {
+		if strings.Index(certIssuer, want+"/") == 0 {
+			return fmt.Errorf("expected Issuer to be %s, got %s -- if you have customized the issuer for your enterprise try using the --cert-oidc-issuer flag with your expected issuer", want, certIssuer)
```

Comment:
small tweak to align it more with the language used in the docs:

```suggestion
			return fmt.Errorf("expected Issuer to be %s, got %s -- if you have a custom OIDC issuer policy for your enterprise, use the --cert-oidc-issuer flag with your expected issuer", want, certIssuer)
```

on the theory that googling `custom OIDC issuer policy for your enterprise` returns the REST API docs & the security hardening with openID connect doc pages. i'm sure the old text also brought those docs pages up but this felt slightly more consistent.